doc/produs_unterlagen/materials/java_workshop.md

```diff
@@ -0,0 +1,91 @@
+# Java Dungeon Workshop
+
+## Zielgruppe und Kurzbeschreibung des Inhalts
+
+Der Workshop „Java Dungeon“ richtet sich an Schüler\:innen ab der 8.–10. Klasse, die grundlegende Informatikkenntnisse mitbringen, aber noch keine oder nur wenig Java-Erfahrung haben. Der Fokus liegt auf einem spielerischen und visuell unterstützten Einstieg in das Programmieren mit Java. Die Teilnehmenden nutzen eine spezielle VS-Code-Extension, um mit Java-Code zu interagieren und ihn im Blockly-Dungeon direkt auszuprobieren.
+
+Ziel ist es, grundlegende Programmierkonzepte wie Variablen, Kontrollstrukturen und Methoden zu verstehen und anzuwenden. Im Zentrum steht dabei das Lösen von Programmier-Rätseln innerhalb einer Dungeon-Spielumgebung.
+
+**Lernform:** Es wird auf exploratives, selbstbestimmtes Arbeiten gesetzt. Die Teilnehmenden erkunden den Dungeon, lösen Rätsel und wählen individuelle Wege zur Lösung. Code-Techniken werden "on demand" erklärt.
+
+## Benötigte Unterlagen und Software
+
+Damit der Workshop reibungslos funktioniert, sind folgende Software-Komponenten und Vorbereitungen notwendig:
+
+### Software-Voraussetzungen
+
+* **Java:** Eine aktuelle Java-Installation ist erforderlich, um das Dungeon-Spiel starten und Java-Code ausführen zu können.
```

Comment:
was genau ist "aktuell"?

gibt es noch die unterscheidung zw. jre und jdk? nicht mehr, oder?

Reply:
Hab ich auch gedacht, aber irgendwie gibts noch beides. Es wird nur als ein Paket vertrieben.
ich hatte die Tage tatsächlich einen Fehler in der richtung: Der Code wurde in einer höheren Java Version gebaut als abgespielt.

Reply:
dann sollte der hinweis auf "jdk 21" tatsächlich mit rein. zumindest als untergrenze. 

andersherum hatte libgdx iirc mit zu neuen java-versionen probleme, aber das könnte auch an der (dann fehlenden) gradle-unterstützung gelegen haben.

Reply:
ich verlink auf die repo md, da steht die benötigte version.